transmart-core-db/src/main/groovy/org/transmartproject/db/tree/TreeNode.groovy

```diff
@@ -176,7 +176,7 @@ class TreeNode {
                         return null
                 }
                 constraint.field = [
-                        dimension: TrialVisitDimension.simpleName,
+                        dimension: 'trial visit',
```

Comment:
This is a dimension, not a constraint. For the dimensions we use instances, so you can't get the name from the class. But I can indeed get the name in a slightly different way. Will fix.